server/app.js

```diff
@@ -56,6 +60,23 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
+// GET request from client > server > S3 bucket
+app.get('/profilePhotos/:key', (req, res) => {
+  console.log(res);
+  const key = req.params.key;
+  console.log(key);
+  const downloadFromS3 = downloadPhoto(key);
+
+  downloadFromS3.pipe(res);
+})
+
+// POST request for uploading to Express server then pushing to S3 Bucket
+app.post('/profilePhotos', upload.single('profilePhoto'), async (req, res) => {
+  const file = req.file;
```

Comment:
First, validate `file` in the request so that if it's missing, an error status should be returned to the client.

Reply:
I see, have a conditional if statement to check for whether or not the file is undefined?

Reply:
Yes, that is correct.